test/clearingHouse/ClearingHouse.badDebt.test.ts

```diff
@@ -56,30 +61,44 @@ describe("ClearingHouse badDebt", () => {
         await collateral.mint(bob.address, parseUnits("100", decimals))
         await deposit(bob, vault, 100, collateral)
 
+        // prepare collateral for carol
```

Comment:
It seems that we still can use Alice, no need Carol, right?

Reply:
Yes, we can use Alice here. But to make tests clearer, would like to specify Alice is maker, and Bob and Carol are takers.